### ISSUE TITLE
Progress bar without colors in companion app, after 2023.12 Upgrade

### ISSUE THREAD

**Issue by BunpGhost** [custom-cards/bar-card#177]
I use colors and gradient in bar card and after the last companion app upgrade (2023.12) it stopped working. 
Now it is all blue :) 

However, in browser it works as always.
Am I doing something wrong or ??

**Comment by midadami** [comment]
Same problem. linear-gradient not working
```
  bar-card-backgroundbar {
    background: linear-gradient(to right, red 10%, orange 20%, green 30%, green 80%, orange 95%,red 97%);
  }
  bar-card-currentbar {
    background: linear-gradient(to right, red 10%, orange 20%, green 30%, green 80%, orange 95%,red 97%);
    clip-path: polygon(0 0, var(--bar-percent) 0, var(--bar-percent) 100%, 0 100%);
  }  
```

**Comment by midadami** [comment]
> I use colors and gradient in bar card and after the last companion app upgrade (2023.12) it stopped working. Now it is all blue :)
> 
> However, in browser it works as always. Am I doing something wrong or ??

I had same problem. Check if you using the new implementation of Card-mod released on 3.4.2. That fixed it for me.

**Comment by BunpGhost** [comment]
Yes, you are right. But I also had to change the code like so:
`card_mod:
              style: |-
                bar-card-currentbar {
                  background: linear-gradient(to right,{% if is_state("sun.sun", "above_horizon") -%}lightblue 0%,{%- else -%}blue 0%,{%- endif %}orange 100%);
                  clip-path: polygon(0 0, var(--bar-percent) 0, var(--bar-percent) 100%, 0 100%);
                }`

I'll close the issue!
